internal/pkg/ecs/ecs.go

```diff
@@ -232,6 +246,37 @@ func (c Client) StopDefaultClusterTasks(familyName string) error {
 	return c.ecsClient.StopTasks(taskIDs, ecs.WithStopTaskReason(taskStopReason))
 }
 
+// DecryptedSecrets returns the decrypted parameters from either the SSM store or Secrets Manager.
+func (c Client) DecryptedSecrets(secrets []*ecs.ContainerSecret) ([]EnvVar, error) {
+	var ssmSecrets []EnvVar
+	var secretManagerSecrets []EnvVar
```

Comment:
could we just have a single list? seems like they just get combined anyways...
```suggestion
	var vars []EnvVar
```